device/ipv4.yaml

```diff
@@ -36,3 +34,31 @@ components:
             The IPv4 address of the gateway
           type: string
           format: ipv4
+          default: 0.0.0.0
```

Comment:
If a property is made as required, no need to add defaults. Please refer to this earlier PR https://github.com/open-traffic-generator/models/pull/125

Reply:
done